src/circuits/query.ts

```diff
@@ -1,8 +1,10 @@
-import keccak256 from 'keccak256';
 import { ISchemaLoader, SchemaLoadResult } from '@lib/loaders/schema';
 import nestedProperty from 'nested-property';
-import { Schema } from '@lib/protocol/models';
-import { fromLittleEndian } from '@lib/core/util';
+import { Id, SchemaHash } from '@iden3/js-iden3-core';
+import { getContextPathKey } from '@lib/merklize/path';
+import { TextDecoder } from 'util';
```

Comment:
TextDecoder and TextEncoder should be accesible without any imports, so you could remove them